patch-testing/secp256k1/src/lib.rs

```diff
@@ -1,10 +1,21 @@
-#[cfg(test)]
 use secp256k1::{Message, PublicKey, Secp256k1};
 
 #[sp1_test::sp1_test("secp256k1_recover", syscalls = [SECP256K1_DOUBLE, SECP256K1_ADD])]
 fn test_recover_rand_lte_100(
     stdin: &mut sp1_sdk::SP1Stdin,
 ) -> impl FnOnce(sp1_sdk::SP1PublicValues) {
+    recover_rand_lte_100(stdin)
+}
+
+#[sp1_test::sp1_test("secp256k1_recover_v0_30_0", syscalls = [SECP256K1_DOUBLE, SECP256K1_ADD])]
+fn test_recover_v0_30_0_rand_lte_100(
+    stdin: &mut sp1_sdk::SP1Stdin,
+) -> impl FnOnce(sp1_sdk::SP1PublicValues) {
+    recover_rand_lte_100(stdin)
+}
+
+#[allow(dead_code)]
```

Comment:
Why is this `dead_code` necessary? Because the functions are tests?

Reply:
Exactly